Week2/homework/masoud/pokemon-app/script.js

```diff
@@ -0,0 +1,71 @@
+//Calling main function after window load
+window.onload = main;
+
+//Define main function
```

Comment:
You don't need comments for things that are obvious.  e.g. it is obvious that you're defining the main function.
Sometimes you might do something which is not clear if someone else is going to read it. That is an example where it is good to include comments.